SoftLayer/CLI/firewall/add.py

```diff
@@ -15,7 +15,7 @@
               type=click.Choice(['vs', 'vlan', 'server']),
               help='Firewall type',
               required=True)
-@click.option('--high-availability', '--ha',
+@click.option('--high-availability', '-ha',
```

Comment:
`--ha` is only two characters longer and holds much more information about what it means than just 'h'. Plus, just swapping the two doesn't present an interface change so if any scripts did exist which rely on `--ha` then they would not need to change. For those reasons I prefer just swapping the two arguments (`@click.option('--ha', '--high-availability',`) and not changing the outward interface in any way.